yoeo/detect.py

```diff
@@ -26,9 +28,9 @@
 from matplotlib.ticker import NullLocator
 
 
-def detect_directory(model_path, weights_path, img_path, classes, output_path,
+def detect_directory(model_path, weights_path, img_path, class_config: ClassConfig, output_path,
```

Comment:
We could maybe make the class config optional and no grouping etc. is done if it is not provided

Reply:
_ClassConfig_ != _GroupConfig_ . Optional doesn't work with _ClassConfig_ because it provides the class names even if **no** grouping is to be applied. So a _classs_config_ **must** always exist. However, _GroupConfig_ is optional works and exactly as you suggested.

Reply:
Ah okay this makes sense